apps/trading-e2e/src/integration/withdrawals.feature

```diff
@@ -0,0 +1,67 @@
+Feature: Withdrawals
+
+  Scenario: Can prepare a withdrawal
+    Given I navigate to withdrawals page
+    And I connect eth wallet
+    And I connect vega wallet
+    And I can see the withdrawals warning info
```

Comment:
I think we should remove this step and instead have an additional scenario where the user has incomplete withdrawals.

It would look something like:
```
Given I am on the create withdrawal page
And I have incomplete withdrawals
Then I can navigate to the withdrawals page
```

Reply:
we already have such tests towards the bottom . in this scenario we are just checking we can prepare a withdrawal rather than complete it (phase 1)